plugin-server/src/worker/ingestion/hooks.ts

```diff
@@ -203,7 +202,7 @@ export class HookCommander {
             await Promise.all(webhookRequests).catch((error) => captureException(error))
         }
 
-        if (organization!.available_features.includes('zapier')) {
+        if (await this.organizationManager.hasAvailableFeature(team.id, 'zapier', team)) {
```

Comment:
This is the reason why the cache expiration on plan upgrade is relevant in this PR, isn't it?

Reply:
Fair. I'll revert this part + create a task to follow-up this.